StacksOfPlates.py

```diff
@@ -0,0 +1,48 @@
+class Stack:
+    def __init__(self, cap):
+        self.stack = []
+        self.cap = cap
+
+    def isFull(self):
+        return len(self.stack) == self.cap
+
+    def push(self, item):
+        if not self.isFull():
+            self.stack.append(item)
+
+    def pop(self):
+        if self.stack:
+            return self.stack.pop()
+
+class SetOfStacks:
+    def __init__(self, stackCap):
+        self.stacks = []
+        self.stackCap = stackCap
+
+    def push(self, item):
+        if not self.stacks or self.stacks[-1].isFull():
+            newStack = Stack(self.stackCap)
+            newStack.push(item)
+            self.stacks.append(newStack)
+        else:
+            self.stacks[-1].push(item)
+
+    def pop(self):
+        popped = self.stacks[-1].pop()
+        if not self.stacks[-1].stack:
+            self.stacks.pop()
+        return popped
```

Comment:
This would be cleaner if you changed the order of operations here. Because you're only using `popped` once, it makes more sense to skip declaring this variable, but since you need to make sure you're not popping from an empty stack, you can do the conditional check first.
```suggestion
        if not self.stacks[-1].stack:
            self.stacks.pop()
        return self.stacks[-1].pop()
```